plugins/cloudinary/package.json

```diff
@@ -24,9 +24,9 @@
     "prebuild": "rimraf dist",
     "build": "tsc --module commonjs && rollup -c rollup.config.ts",
     "release:dev": "npm run build && npm version prerelease --no-git-tag-version && npm publish --tag dev",
-    "start": "SERVE=true rollup -c rollup.config.ts -w",
-    "test": "jest --coverage",
-    "test:watch": "jest --coverage --watch",
+    "start": "cross-env SERVE=true rollup -c rollup.config.ts -w",
```

Comment:
This makes sure environment variable is set regardless of the OS (windows, linux, etc...)